css/style.css

```diff
@@ -1,19 +1,157 @@
+/* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
 
- /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
+body, footer {
+  font-family: "Roboto", sans-serif;
+  -webkit-font-smoothing: antialiased;
+}
+
+/*Navigation*/
+.navbar{
+  background-color: white;
+  position: sticky;
+}
+.navbar .logo {
+  height: 60px;
+  margin-left: 30px;
+  float: left;
+}
+.navbar a{
+  float: left;
+  margin-top: 0;
+  display: block;
+  color: grey;
+  text-align: center;
+  padding: 14px 20px;
+  background-color: white;
+}
+.navbar a:hover{
+  color: orange;
+}
+.right {
+  float: right;
+}
+a{
+  text-decoration: none;
+}
+.navbar a.current-tab{
+  color: black;
+}
+.navbar .icon {
+  display: none;
+}
+
+
+/*Introduction General styles*/
+.intro{
+  background-image: url(/img/first-background.jpg);
+  background-size: cover;
+	background-repeat: no-repeat;
+	background-position: center bottom;
+  width: 100%;
+  height: auto;
+  overflow: auto;
+}
+  
+.intro .content{
+  color: white;
+  text-align: center;
+  padding: 100px;
+  padding-top: 0px;
+}
+.intro h1{
+  font-size: 50px;
+  margin-top: 200px;
+  font-weight: 100;
+}
+.intro h3{
+  font-size: 20px;
+  margin-top: 20px;
+  font-weight: 100;
+}
+
+/*Button - general styles*/
+.button{
+  background-color: orange;
+  color: white;
+}
+.button:hover{
+  color: red;
+}
+
+/*Main text of the landing page - General styles*/
+.main{
+  display: flex;
+  justify-content: center;
+  margin-left: 20px;
+  margin-right: 20px;
+  vertical-align: middle;
+  flex-direction: row;
+  
+}
+/*Icons styles*/
+.main .devices{
+  width: 100%;
+  height: auto;
+  padding: 115px;
+  margin-top: 0px;
+}
+.main .coffee {
+  width: 100%;
+  height: auto;
+  padding:115px;
+  margin-top: 0px;
+}
+.main .refill{
+  width: 100%;
+  height: auto;
+  padding:115px;
+  margin-top: 0px;
```

Comment:
Remember, the purpose of CSS classes is that they are re-usable! 
Don't repeat your code here. Only define these styles once for one class (for example `.main-icon`) and then apply that class to all 3 elements in your HTML.

Reply:
Updated CSS